Apps/Medication/src/Services/RestMedicationStatementService.cs

```diff
@@ -83,7 +83,8 @@ private async Task<HNMessage<List<MedicationStatement>>> RetrieveMedicationState
         {
             string jwtString = this.httpContextAccessor.HttpContext.Request.Headers["Authorization"][0];
             string phn = await this.patientDelegate.GetPatientPHNAsync(hdid, jwtString).ConfigureAwait(true);
-            string userId = this.httpContextAccessor.HttpContext.User.Identity.Name;
+            //string userId = this.httpContextAccessor.HttpContext.User.FindFirst("hdid").Value;
+            string userId = "USER_ID";
```

Comment:
did you leave this on purpose? if so lets create task to fix it in the future

Reply:
Let's create a common function to string build a 'surname.given_names' as userId then prune it to max length in the field setter downstream.

Reply:
Yep, I've created an issue for that https://qslvic.visualstudio.com/Health%20Gateway/_workitems/edit/7606